### PR TITLE
MediaTransportElement Background Fix

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

- Added `MediaTransportControls` test page
- Updated `MediaTransportControlsPanelBackground` to `ControlOnImageFillColorDefaultBrush`

## How Has This Been Tested?

Visual verification

## Screenshots (if appropriate):

image
image

### CODE REVIEW

File: dev/dll/Microsoft.UI.Xaml.vcxproj [hunk 1]
```diff
@@ -617,7 +618,7 @@
       <PRIResource Remove="@(PRIResource)" />
     </ItemGroup>
     <!-- Use CreateItem to expand all the ** to individual full paths -->
-    <CreateItem Include="@(PRIResourceExpanded -> '%(Replaced)')">
+    <CreateItem Include="@(PRIResourceExpanded -&gt; '%(Replaced)')">
```
Comment (beervoley) [inline dev/dll/Microsoft.UI.Xaml.vcxproj:621]: **-&gt** what is this change?:) and it's repeating below several times

Comment (ranjeshj) [reply dev/dll/Microsoft.UI.Xaml.vcxproj:621]: Visual studio tends to do this unfortunately.

---
In reply to: [587828088](https://github.com/microsoft/microsoft-ui-xaml/pull/4403#discussion_r587828088) [](ancestors = 587828088)

**Comment by ranjeshj** [comment]
>     WriteLine("");

no change, please revert this file.

---
Refers to: dev/dll/XamlMetadataProviderGenerated.tt:82 in c2588ff. [](commit_id = c2588ffed19d37c6f12873359090c305c2ffa8cd, deletion_comment = False)

File: dev/MediaTransportControls/InteractionTests/MediaTransportControlsTests.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            TestEnvironment.Initialize(testContext);
+        }
+
+        public void TestCleanup()
+        {
+            TestCleanupHelper.Cleanup();
+        }
+
+        [TestMethod]
+        public void BasicTest()
```
Comment (ranjeshj) [inline dev/MediaTransportControls/InteractionTests/MediaTransportControlsTests.cs:46]: The entire mediacontrolstransportcontrols project seems mostly empty. Since you are just adding the resources to common styles, can you just add your test ui to the common styles test pages ? That will avoid adding a bunch of these unsed projects.

File: dev/dll/Microsoft.UI.Xaml.vcxproj (hunk 1 above)
Comment (karenbtlai) [reply dev/dll/Microsoft.UI.Xaml.vcxproj]: reverted

File: dev/Telemetry/RuntimeProfiler.h [hunk 3]
```diff
@@ -55,6 +55,7 @@ namespace RuntimeProfiler
         ProfId_PipsPager,
         ProfId_ImageIcon,
         ProfId_AnimatedIcon,
+        ProfId_MediaTransportControls,
```
Comment (ranjeshj) [inline dev/Telemetry/RuntimeProfiler.h:58]: we can remove this

File: MUXControls.sln [hunk 4]
```diff
@@ -719,6 +719,7 @@ Global
 		dev\Materials\Acrylic\APITests\AcrylicBrush_ApiTests.projitems*{593e15d8-f6f9-4aba-ba65-c6927c178dbd}*SharedItemsImports = 13
 		dev\Materials\Reveal\TestUI\Reveal_TestUI.projitems*{5bf80ae9-29df-4be9-858a-f095c8073473}*SharedItemsImports = 13
 		dev\PersonPicture\PersonPicture.vcxitems*{5fdf2501-aa3d-4082-ad45-d1f2a94c1213}*SharedItemsImports = 9
+		dev\ScrollBar\TestUI\ScrollBar_TestUI.projitems*{6075e2d2-46d6-4000-bb77-56f0efe37389}*SharedItemsImports = 13
```
Comment (ranjeshj) [inline MUXControls.sln:722]: this seems unrelated

File: MUXControlsInnerLoop.sln [hunk 5]
```diff
@@ -478,6 +478,16 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AnimatedIcon", "dev\Animate
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "AnimatedIcon_TestUI", "dev\AnimatedIcon\TestUI\AnimatedIcon_TestUI.shproj", "{83FD36C0-189F-4E95-8002-9B73905CA301}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "MediaTransportControls", "MediaTransportControls", "{D61A3A5B-3DAD-4BC4-94C5-618AD02F946A}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "MediaTransportControls", "dev\MediaTransportControls\MediaTransportControls.vcxitems", "{AE1A4B23-53BD-4170-87E6-3507F9F4862F}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "MediaTransportControls_APITests", "dev\MediaTransportControls\APITests\MediaTransportControls_APITests.shproj", "{639E0316-C106-4222-952B-8725CE4531EE}"
+EndProject
```
Comment (ranjeshj) [inline MUXControlsInnerLoop.sln:486]: we could remove these as well in this file.

File: test/MUXControls.Test/MUXControls.Test.Shared.targets [hunk 6]
```diff
@@ -42,4 +42,5 @@
   <Import Project="..\..\dev\PagerControl\InteractionTests\PagerControl_InteractionTests.projitems" Label="Shared" Condition="$(FeaturePagerControlEnabled) == 'true'" />
   <Import Project="..\..\dev\PipsPager\InteractionTests\PipsPager_InteractionTests.projitems" Label="Shared" Condition="$(FeatureNumberBoxEnabled) == 'true'" />
   <Import Project="..\..\dev\Breadcrumb\InteractionTests\Breadcrumb_InteractionTests.projitems" Label="Shared" Condition="$(FeatureBreadcrumbEnabled) == 'true'" />
+  <Import Project="..\..\dev\MediaTransportControls\InteractionTests\MediaTransportControls_InteractionTests.projitems" Label="Shared" Condition="$(FeatureMediaTransportControlsEnabled) == 'true'" />
```
Comment (ranjeshj) [inline test/MUXControls.Test/MUXControls.Test.Shared.targets:45]: remove ?

File: test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems [hunk 7]
```diff
@@ -146,6 +146,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)Assets\ladybug.wmv" />
```
Comment (StephenLPeters) [inline test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems:149]: is this asset okay to use copywrite wise?

File: test/MUXControlsTestApp/MUXControlsTestApp.Shared.targets [hunk 8]
```diff
@@ -84,4 +84,6 @@
   <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\ImageIcon\APITests\ImageIcon_APITests.projitems" Label="Shared" Condition="$(FeatureImageIconEnabled) == 'true'" />
   <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\AnimatedIcon\TestUI\AnimatedIcon_TestUI.projitems" Label="Shared" Condition="$(FeatureAnimatedIconEnabled) == 'true'" />
   <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\AnimatedIcon\APITests\AnimatedIcon_APITests.projitems" Label="Shared" Condition="$(FeatureAnimatedIconEnabled) == 'true'" />
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\MediaTransportControls\TestUI\MediaTransportControls_TestUI.projitems" Label="Shared" Condition="$(FeatureMediaTransportControlsEnabled) == 'true'" />
```
Comment (StephenLPeters) [inline test/MUXControlsTestApp/MUXControlsTestApp.Shared.targets:87]: these can be reverted right?

File: test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems (hunk 7 above)
Comment (ranjeshj) [reply test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems:149]: This is the same we use in XCG.

**Review by ranjeshj — APPROVED** [review]

File: test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems (hunk 7 above)
Comment (karenbtlai) [reply test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems:149]: Yup, this is where I grabbed it.
